sp800-63c/sec5_assertions.md

```diff
@@ -79,13 +83,17 @@ Assertions SHALL use audience restriction techniques to allow an RP to recognize
 
 In some circumstances, it is desirable to prevent the subscriber's account at the IdP from being easily linked at multiple RPs through use of a common identifier. In these circumstances, pairwise pseudonymous subject identifiers SHALL be used within the assertions generated by the IdP for the RP, and the IdP SHALL generate a different identifier for each RP as described in [Section 5.2.6](#ppi-gen) below.
 
+>**MG: Any concern that "in some circumstances" with a SHALL is too weak?**
```

Comment:
I had previously argued for a global "SHOULD" or "MAY" here but a conditional "SHALL" was the compromise. I am against a global "SHALL" due to the limitations discussed in the following paragraphs.